server/routerlicious/packages/lambdas/src/deli/lambdaFactory.ts

```diff
@@ -164,14 +164,39 @@ export class DeliLambdaFactory extends EventEmitter implements IPartitionLambdaF
             this.serviceConfiguration,
             sessionMetric,
             sessionStartMetric);
+
+        deliLambda.on("close", (closeType) => {
+            const handler = async () => {
+                if ((closeType === LambdaCloseType.ActivityTimeout || closeType === LambdaCloseType.Error)) {
+                    const result = await this.collection.findOne({ documentId });
+                    const sessionP = result?.session;
```

Comment:
This is not a promise right? We don't need the P suffix

Reply:
Thanks. Done. :)